go/flags/endtoend/vtgate.txt

```diff
@@ -1,6 +1,9 @@
 Usage of vtgate:
       --allowed_tablet_types TabletTypeList                              Specifies the tablet types this vtgate is allowed to route queries to
       --alsologtostderr                                                  log to standard error as well as files
+      --balancer_enabled                                                 Whether to enable the tablet balancer to evenly spread query load
```

Comment:
Note for later: "balancer" feels pretty vague to me (there are several concepts of "balancing in Vitess)

Reply:
Good point. I could change this to `tabletbalancer_*` although it is longer than I like. Or maybe wait for suggestions when this is upstreamed and make a PR to sync with that?